OpenOversight/tests/conftest.py

```diff
@@ -348,15 +381,23 @@ def add_mockdata(session):
 
     # assures that there are some assigned and unassigned images in each department
     assigned_images_dept1 = models.Image.query.filter_by(department_id=1).limit(3).all()
-    assigned_images_dept2 = models.Image.query.filter_by(department_id=2).limit(2).all()
+    assigned_images_dept2 = models.Image.query.filter_by(department_id=2).limit(3).all()
 
     jobs_dept1 = models.Job.query.filter_by(department_id=1).all()
     jobs_dept2 = models.Job.query.filter_by(department_id=2).all()
+
+    assignment_ratio = 0.9
     assignments_dept1 = [
-        build_assignment(officer, test_units, jobs_dept1) for officer in officers_dept1
+        build_assignment(officer, test_units, jobs_dept1)
+        for officer in officers_dept1[
+            : math.ceil(len(officers_dept1) * assignment_ratio)
+        ]
     ]
     assignments_dept2 = [
-        build_assignment(officer, test_units, jobs_dept2) for officer in officers_dept2
+        build_assignment(officer, test_units, jobs_dept2)
+        for officer in officers_dept2[
+            : math.ceil(len(officers_dept2) * assignment_ratio)
+        ]
```

Comment:
🔬  The syntax here doesn't feel incredibly intuitive.

Reply:
Tried to make it a little better, by extracting "the math" to a variable and adding a comment.